libs/content/workspace/src/space_inspector/show.rs

```diff
@@ -445,7 +450,33 @@ impl SpaceInspector {
                 if response.clicked() && item_filerow.file.is_folder() {
                     changed_focused_folder = Some(item.id);
                 }
+                // Context menu
+                response.context_menu(|ui| {
+                    ui.spacing_mut().button_padding = egui::vec2(4.0, 4.0);
+
+                    ui.label(&self.data.all_files[&item.id].file.name.to_string());
+                    ui.label(&display_size);
+                    ui.separator();
 
+                    if ui.ctx().input(|i| i.key_pressed(egui::Key::Escape)) {
+                        ui.close_menu();
+                    }
+                    if item_filerow.file.is_folder() {
+                        if ui.button("Make Root").clicked() {
+                            changed_focused_folder = Some(item.id);
+                            ui.close_menu();
+                        }
+                    }
+
+                    if ui.button("Delete").clicked() {
+                        let lb = self.lb.clone();
+                        let id = item_filerow.file.id;
+                        deleted_id = Some(id);
+                        thread::spawn(move || {
+                            let _ = lb.delete_file(&id);
```

Comment:
I'd like to see this error handled before merging -- you can unwrap, it's within a thread and the application won't die. Unwrapping will at-least generate log and panic files. Or you can handle it better.